shared/provision/username-or-email/container.js

```diff
@@ -24,5 +27,11 @@ export default compose(
     dispatchToProps,
     (s, d, o) => ({...o, ...s, ...d})
   ),
-  safeSubmit(['onBack', 'onSubmit'], ['error'])
+  withStateHandlers(
```

Comment:
should likely not use recompose and just move this state into the component. its not used at all by this container